packages/teams-js/src/public/meeting.ts

```diff
@@ -210,6 +210,36 @@ export namespace meeting {
     error?: SdkError;
   }
 
+  /**
+   * Interface for audio handling response
+   *
+   * @beta
+   */
+  export interface IRequestAppAudioHandlingSdkResponse {
+    /** Indicates whether host should handle audio*/
+    isHostAudioless: boolean;
+  }
+
+  /**
+   * Interface for mic state change
+   *
+   * @beta
+   */
+  export interface IMicState {
+    /** Indicates the mute status of the mic*/
+    isMicMuted?: boolean;
+
+    /**
+     * error object in case there is a failure
+     */
+    error?: SdkError;
+  }
+
+  export interface RequestAppAudioHandlingParams {
+    isAppHandlingAudio: boolean;
+    callbackMicMuteStateChangedHandler: (micState: IMicState) => void;
```

Comment:
Rather than having this interface return a nullable boolean and a nullable error to the user, it would probably be better to just have it always return isMicMuted to the user and then throw any error it gets from the host